docs/pages/includes/kubernetes-access/rbac.mdx

```diff
@@ -1,21 +1,22 @@
-To use Teleport to interact with a Kubernetes cluster, your Teleport roles must
-allow access from your Kubernetes user and groups. Ensure that you have a
-Teleport role that grants access to the cluster you plan to interact with.
+### Kubernetes Authentication
```

Comment:
I guess I missed this one. I don't think we should include this partial at all, since the guide already explains setting up a `tadmin` user with the `system:masters` kube group.

I double-checked the other places we use this partial and I don't see issues with the others.

Reply:
That sounds good to me!